### PR TITLE
Updated model and api according to new DB

### PR DESCRIPTION
**Pull request by vishakhatak**

### CODE REVIEW

File: OnlineAssetManagementTool/OnlineAssetManagementTool.API/Controllers/API/Persistence/AssetAssignmentController.cs [hunk 1]
```diff
@@ -24,11 +24,11 @@ public AssetAssignmentController()
 
         [HttpPost]
         [Route("create")]
-        public async Task<IHttpActionResult> CreateNewAssetAssignmentAsync([FromBody] Asset_Assignment newAssetAsignment)
+        public async Task<IHttpActionResult> CreateNewAssetAssignmentAsync([FromBody] Asset_Assignment newAssetAssignment)
         {
             try
             {
-                var result = await Task.Run(() => assetAssignmentRepository.CreateNewAssetAssignemnt(newAssetAsignment));
+                var result = await Task.Run(() => assetAssignmentRepository.CreateNewAssetAssignemnt(newAssetAssignment));
```
Comment (sakshiITT) [inline OnlineAssetManagementTool/OnlineAssetManagementTool.API/Controllers/API/Persistence/AssetAssignmentController.cs]: Why "new" word being added in newAssetAssignment?

File: OnlineAssetManagementTool/OnlineAssetManagementTool.DAL/Repositories/Persistence/AssetAssignmentRepository.cs [hunk 2]
```diff
@@ -33,12 +33,13 @@ public bool DeleteAssetAsignment(int id)
             bool isDeleted = false;
             Asset_Assignment assetAssignmentToDelete = AMSDb.Asset_Assignment.FirstOrDefault(u => u.Id == id);
             assetAssignmentToDelete.Is_Assigned = "False";
+            assetAssignmentToDelete.Asset.Is_Assigned = "False";
```
Comment (sakshiITT) [inline OnlineAssetManagementTool/OnlineAssetManagementTool.DAL/Repositories/Persistence/AssetAssignmentRepository.cs]: Cant it be assignedAssetToDelete?

Comment (vishakhatak) [reply OnlineAssetManagementTool/OnlineAssetManagementTool.DAL/Repositories/Persistence/AssetAssignmentRepository.cs]: Changing to this name will change the meaning of the operation.

File: OnlineAssetManagementTool/OnlineAssetManagementTool.API/Controllers/API/Persistence/AssetAssignmentController.cs (hunk 1 above)
Comment (vishakhatak) [reply OnlineAssetManagementTool/OnlineAssetManagementTool.API/Controllers/API/Persistence/AssetAssignmentController.cs]: Have added a task for refactoring naming conventions in this file.

**Pull request merged**